honestcash-v2/src/app/modules/editor/components/editor/editor.component.ts

```diff
@@ -120,14 +120,7 @@ export class EditorComponent implements OnInit, OnDestroy {
   }
 
   public downloadImageFromUrlAndUpload(url: string) {
-    return this.editorService.downloadImageFromUrl(url).toPromise().then(response => {
-      return {
-        success: 1,
-        file: {
-          url: response.files[0].url
-        }
-      };
-    });
+    return this.editorService.uploadRemoteImage(url).toPromise();
```

Comment:
you can't simply do ".subscribe()" ?

Reply:
no because this is mapped to editorjs configuration, it expects promise